milli/src/search/facet/filter.rs

```diff
@@ -39,12 +42,24 @@ impl<'a> std::error::Error for FilterError<'a> {}
 impl<'a> Display for FilterError<'a> {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            Self::AttributeNotFilterable { attribute, filterable } => write!(
-                f,
-                "Attribute `{}` is not filterable. Available filterable attributes are: `{}`.",
-                attribute,
-                filterable,
-            ),
+            Self::AttributeNotFilterable { attribute, filterable_fields } => {
+                let filterable = filterable_fields.iter().map(AsRef::as_ref).collect::<Vec<_>>().join(" ");
+
+                if filterable.is_empty() {
+                    write!(
+                        f,
+                        "Attribute `{}` is not filterable. This index does not have configured filterable attributes.",
+                        attribute,
+                    )
+                } else {
+                    write!(
+                        f,
+                        "Attribute `{}` is not filterable. Available filterable attributes are: `{}`.",
+                        attribute,
+                        filterable,
+                    )
+                }
```

Comment:
I think so @Kerollmops, because without it I receive this error in the `join()`:

<img width="675" alt="image" src="https://user-images.githubusercontent.com/4116980/158598448-50fad94f-d565-40d2-9b75-0cde2d9ca132.png">

<img width="616" alt="image" src="https://user-images.githubusercontent.com/4116980/158598649-38ad54cd-0664-429e-a9ab-6b8d9e13b987.png">

Reply:
Ok, so you must keep it then 😄